api/src/main/java/org/pmiops/workbench/db/dao/GoogleProjectPerCostDao.java

```diff
@@ -0,0 +1,15 @@
+package org.pmiops.workbench.db.dao;
+
+import org.pmiops.workbench.db.model.DbGoogleProjectPerCost;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+public interface GoogleProjectPerCostDao extends CrudRepository<DbGoogleProjectPerCost, Long> {
+
+  @Query(
```

Comment:
do you need to write this query? Wouldn't JPA can take care if it. 
Also, suggest to write a unit test for Dao then you can easily find the result. 
https://github.com/all-of-us/workbench/tree/03138d9237d197a16da91fb3e9e908d5a46aca0b/api/src/test/java/org/pmiops/workbench/db/dao